RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -4,6 +4,7 @@
 
 #for dnn classifier
 from Configuration.ProcessModifiers.trackdnn_cff import trackdnn
+from dnnQualityCuts import qualityCutDictionary
```

Comment:
I think we have a convention of using full import paths, i.e.

```python
from  RecoTracker.IterativeTracking.dnnQualityCuts import qualityCutDictionary
```

This comment applies everywhere where this is imported.

Reply:
as it is, this import also won't work with python3.